examples/js/loaders/AMFLoader.js

```diff
@@ -48,18 +48,18 @@ THREE.AMFLoader.prototype = {
 
 	parse: function ( data ) {
 
-		function loadDocument( data ) {
+		function _loadDocument( data ) {
```

Comment:
Mostly because flat helps show the faces so that someone reviewing the model for printing can easily see the faces.

If I wanted to allow users to specify a default material would it be better to make a param or use a public member?